docs/releases/1.3.txt

```diff
@@ -314,7 +314,7 @@ requests. These include:
 * Support for combining :class:`F expressions <django.db.models.F>`
   with ``timedelta`` values when retrieving or updating database values.
 
-.. _HttpOnly: https://www.owasp.org/index.php/HttpOnly
+.. _HttpOnly: https://owasp.org/www-community/HttpOnly
```

Comment:
Don't know whether it is needed or not?

Reply:
It's fine to avoid redirection. Please update the same link in `ref/request-response.txt`.